RubyConf-NG/openemrs
# Develop

#### What does this PR do?
This PR upgrades the Ruby and Rails versions and also adds webpacker.

#### Description of Task to be completed?

1. Upgrading the rails version from 6.0.2 to 6.1.0
2. Upgrading the ruby version from 2.5.3 to 2.7.2
3. Adding support for .env file for the database credentials
4. Organizing the Gemile
5. Adding webpacker and procfiles for on-the-fly assets-precompilation.
6. Upgrading some gems in the Gemfile as well.

#### How should this be manually tested?
You can check the `Gemfile` for the upgrades done.
You can also check the `config/database.yml` file for the modified database environment variables
You will need to create a `.env `file to use in your development environment that will contain your database environment values.

#### Any background context you want to provide?

More background context will be provided in the updated `README.md` file that will be added soon.
